src/components/Comments/Comments.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react'
+import { CommentCard } from '../CommentCard/CommentCard'
+import { getCommentsById } from '../../../APIs'
+
+export const Comments = ({ article_id }) => {
+
+    const [comments, setComments] = useState([])
```

Comment:
Don't forget to add loading handling for improved UX wherever fetching/rendering data